README.md

```diff
@@ -0,0 +1,19 @@
+## Binnie mods
+
+All builds can be found on [Jenkins](https://ci.bymarcin.com/job/Binnie-Mods-MC1.11.2/)
```

Comment:
The state of 1.11.2 is buggy and incomplete. This line should be removed, or a disclaimer added.